dist/hijax.js

```diff
@@ -79,13 +79,14 @@
 
             responseHeadersString = xhr.getAllResponseHeaders();
 
-            if (!this.responseHeaders) {
-                this.responseHeaders = {};
-                while ((match = rHeaders.exec(responseHeadersString))) {
-                    this.responseHeaders[match[1].toLowerCase()] = match[2];
-                }
+            // We need to clear the responseHeaders object every time, otherwise
```

Comment:
These comments appear here in `dist/` but not in the `src/` code. They should probably be moved to `src/` and `dist/` should be re-built.